frictionless/eia860m_raw.py

```diff
@@ -0,0 +1,49 @@
+# -*- coding: utf-8 -*-
+"""Datapackage details specific to the EIA Form 860 archives."""
+
+from . import core
+from . import licenses
+from . import contributors
+
+eia860m_raw = {
+    "name": "pudl-raw-eia860m",
+    "title": "PUDL Raw EIA Form 860M",
+    "description":
+        "US Energy Information Administration (EIA) Form 860 M data for "
+        "electric power plants with 1 megawatt or greater combined nameplate "
+        "capacity. Preliminary Monthly Electric Generator Inventory (based on "
+        "Form EIA-860M as a supplement to Form EIA-860)",
+    "profile": "data-package",
+    "keywords": [
+        "preliminary", "year to date", "electricity", "electric", "boiler",
+        "generator", "plant", "utility", "fuel", "coal", "natural gas",
+        "prime mover", "eia860m", "eia860", "retirement", "capacity",
+        "planned", "proposed", "energy", "hydro", "solar", "wind", "nuclear",
+        "form 860M", "eia", "annual", "gas", "ownership", "steam", "turbine",
+        "combustion", "combined cycle", "eia",
+        "energy information administration", "usa"
+    ],
+    "licenses": [licenses.us_govt, ],
+    "homepage": "https://catalyst.coop/pudl/",
+    "sources": [
+        {
+            "title": "US Energy Information Administration",
+            "path": "https://www.eia.gov/electricity/data/eia860m/"
+        }
+    ],
+    "contributors": [contributors.catalyst_cooperative]
+}
+
+
+def datapackager(dfiles):
+    """
+    Produce the datapackage json for the eia860m archival collection.
+
+    Args:
+        metadata: dict of fixed metadata descriptors
+
+    Returns:
+        dict: fields suited to the frictionless datapackage spec
+        https://frictionlessdata.io/specs/data-package/
```

Comment:
What's up with the `dfiles` argument, but `metadata` in the docstring Args section?

Reply:
good point! this was a copy/paste but I was wondering. I fixed it here and also in the other modules.